contracts/Pool/PoolRegistry.sol

```diff
@@ -276,6 +276,8 @@ contract PoolRegistry is Ownable2StepUpgradeable, AccessControlledV8, PoolRegist
             "PoolRegistry: Market already added for asset comptroller combination"
         );
 
+        require(input.initialSupply > 0, "PoolRegistry: initialSupply is zero");
```

Comment:
I could have used custom error here but require is used in other places in the contract, to maintain consistency I have used require.